code/modules/mob/living/simple_animal/hostile/misc.dm

```diff
@@ -283,4 +307,35 @@
 	emote_hear = list("gronks.")
 	meat_amount = 3
 	mob_size = 20
-	resistance = 15
\ No newline at end of file
+	resistance = 15
+
+/mob/living/simple_animal/hostile/retaliate/tahca
+	name = "tahca"
+	desc = "A cervine that is, surprisingly, native to the planet that resembles a two headed deer. Despite what what one expect, while docile, it is more than willing to fight if attacked. Hunters value \
```

Comment:
```suggestion
	desc = "A cervine creature, surprisingly native to this planet, that resembles a two-headed deer. Despite what one would expect, while docile in nature, it is more than willing to fight if attacked. Hunters value \
```

Reply:
This one is still stretching it. _"Despite what one would expect"_ is almost unnecessary given common sense and cohesion (_"docile in nature"_ already tells you that the creature is tame by default, and the _"while"_ at the start is already foreboding of an exception to the rule about to get mentioned) and could be taken out outright without changing the spirit of the flavor text.